### DISCUSSION TITLE
1.7.3

### DISCUSSION THREAD

**Discussion by xushengfeng** [xushengfeng/eSearch#31] [Announcements]
## 主要更新
- 修复部分系统下ocr错误
- 截屏界面插件
- 自定义用户数据路径

## 新增
- 主要
  - 关联png、jpg、svg打开方式
  - 命令行支持打开文件参数
- 截屏
  - 编辑 序号
  - 编辑 箭头
  - 导入css或js插件
- 设置
  - userdata路径

## 修复
- 主页面
  - 浏览器从最大化恢复后标签页窗口大小未适应
- 截屏
  - 从命令行开启时截屏无效
  - 部分情况下工具栏竖直方向超出屏幕
  - 屏幕缩放时导出错位
  - 部分系统下截屏窗口大小不应被更改
  - 修复只点击不拖动时不正确框选
  - 编辑 新建形状时有时会跳跃
- OCR
  - 离线ocr在部分Windows系统下报错找不到模型
- 以图搜图
  - 使用谷歌识图时报错

## 优化
- 所有
  - 禁止选中图标
- 截屏
  - 提高编辑边框的最大宽度
- 设置
  - ocr自定义路径使用文件选择窗口选择
  - ffmpeg路径使用文件选择窗口选择

**全部更新日志**: https://github.com/xushengfeng/eSearch/compare/1.7.2...1.7.3

<hr /><em>This discussion was created from the release <a href='https://github.com/xushengfeng/eSearch/releases/tag/1.7.3'>1.7.3</a>.</em>

**Comment by jayone-c** [comment]
长截图好像有点问题，截图完要卡一下，然后全屏显示长截图= =，这有点尴尬了~

**Reply by xushengfeng** [reply]
这是已知问题，在1.7.2的发行说明里提到过，还在优化

**Comment by Ashine621** [comment]
你好，我在UBUNTU22.04中使用软件时发现所有的截图均为纯透明，使得所有功能无法正常使用，请问是什么原因呢?

**Reply by xushengfeng** [reply]
Ubuntu默认图形管理器是wayland，由于wayland权限问题，wayland版本不支持截屏。如需使用，请在登录界面右下角选择x window
